scss/form/inputs.scss

```diff
@@ -68,3 +68,49 @@
     }
   }
 }
+
+.select-container {
+  position: relative;
+  display: block;
+  width: 100%;
+  &::before {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    width: 30px;
+    height: 100%;
+    content: "";
+    background: black;
+  }
+  &::after {
+    position: absolute;
+    right: 5px;
+    bottom: -7px;
+    width: 30px;
+    height: 100%;
+    color: white;
+    content: ">";
+    transform: rotate(90deg);
+  }
+  .select {
+    display: block;
+    width: 100%;
+    padding: 0.5rem 0;
+    background: white;
+    border: none;
+    outline: none;
+    @include border-style(#000000, #ffffff);
```

Comment:
please use color variables 🙏 

* #000, black -> `$base-color`
* #fff, white -> `$background-color`

Reply:
Okay 😄 I will fix it later 🙇